src/renderer/i18n/ja.ts

```
@@ -356,7 +356,14 @@
       titleAsc: "タイトル（昇順）",
       titleDesc: "タイトル（降順）",
     },
-    empty: "まだプロジェクトはありません。最初のプロジェクトを作成して始めましょう！",
+    empty: {
+      welcome: "ようこそ、MulmoCast へ！",
+      introduction1: "まずは「{createNew}」ボタンを押してみましょう。",
+      introduction2: "サンプル入りのプロジェクトが自動で用意されます。あとは「{generateVideo}」ボタンを押すだけで体験できます。",
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)